### ISSUE TITLE
observation.space() method when clip is false

### ISSUE THREAD

**Issue by mhtb32** [eleurent/highway-env#88]
In `KinematicsObservation`, the `space()` method always returns a `Box` with range (-1, 1). At the same time, the constructor takes a `clip` argument, which, while false, allows observations to be outside the box and in the range `features_range`(and even outside that in some cases like what we discussed on #80 ).

In those cases, one can not rely on this method to give the right observation space.

**Comment by eleurent** [comment]
Indeed. 

But, though it is definitly good practice, I do not think that openai.gym enforces the observation to belong to the observation space. Thus, When the observation is not clipped, the bounds of the observation space could are not relevant, but could still be useful for indication, to give an expected order of magnitude for instance.

But if you think this is a slippery slope, we could also set the Box bounds to +/- np.inf, and then we'd be fine.

**Comment by mhtb32** [comment]
I confronted with this problem when I set `clip` to `False`, and while running tests, I got assertion error from [this line](https://github.com/eleurent/highway-env/blob/master/tests/envs/test_gym.py#L28) time to time(after every 6 or 7 test runs for example).

But by changing `Box` bounds to infinity, that assertion line becomes meaningless. So after your explanation, maybe it is better to leave everything the way it is now.
